### PR TITLE
Simple action server

### PR DESCRIPTION
**Pull request by disRecord**
In our project we implemented the analog of ROS actionlib::SimpleActionServer. Unfortunately, its interface is different from ROS but it still hides all annoying checks of GoalHandle state.

Repository https://github.com/disRecord/rtt_ros_examples contains working usage examples.

### CODE REVIEW

**Comment by ahoarau** [comment]
Good job ! But there's a tiny typo : `action_server.isPreemting()` should be `action_server.isPreempting()`

File: rtt_actionlib/README.md [hunk 1]
```diff
@@ -138,14 +138,102 @@ some_component_name.actionlib.connectSub(
...
+
+```cpp
+class Component : public TaskContext
+{
+    protected:
+        // Goal, Feedback, Result typedefs
+        ACTION_DEFINITION(ActionSpec);    
+
+    protected:
+        OrocosSimpleActionServer<ActionSpec> action_server;
```
Comment (meyerj) [inline rtt_actionlib/README.md]: -> `RTTSimpleActionServer`

**Comment by disRecord** [comment]
I've fixed the typo. Excuse me for long delay

**Comment by ahoarau** [comment]
@disRecord ping for the headers !

**Comment by disRecord** [comment]
???
What are you meaning?

**Comment by ahoarau** [comment]
The header should be ` #include <rtt_actionlib/rtt_action_server.h>` instead of `#include <orocos/rtt_actionlib/rtt_action_server.h>`

**Comment by ahoarau** [comment]
LGTM ! @meyerj ?

File: rtt_actionlib/include/rtt_actionlib/rtt_simple_action_server.h [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+template <class ActionSpec> class RTTSimpleActionServer 
+{
+	protected:
+		// GoalHandle
+		typedef actionlib::ServerGoalHandle<ActionSpec> GoalHandle;
+		// Goal, Feedback, Result typedefs
+		ACTION_DEFINITION(ActionSpec);	
+
+	protected:
+		rtt_actionlib::RTTActionServer<ActionSpec> action_server;
```
Comment (meyerj) [inline rtt_actionlib/include/rtt_actionlib/rtt_simple_action_server.h]: omit namespace?

**Review by meyerj — APPROVED** [review]
I do not have time to test the patch myself, but from looking at the diff and documentation update it looks good after the comments have been addressed. Thanks for the contribution!

A unit test in [tests/rtt_actionlib_tests](https://github.com/orocos/rtt_ros_integration/tree/toolchain-2.9/tests/) would be great, but we also do not have one for the `RTTActionServer` yet.

**Comment by disRecord** [comment]
I removed unnecessary namespace and fixed typo in README.

**Pull request merged**